.github/workflows/ci-semantic-pull-request.yml

```diff
@@ -61,8 +61,8 @@ jobs:
           # bk      -> bookkeeper
           scopes: |
             admin
+            bk
```

Comment:
@michaeljmarshall I find that you add `bk` and `zk` scopes. There're several overlapping scopes now:

* `zk` vs `meta`
* `bk` vs `ml` vs `storage`

How can you understand this situation?